.github/workflows/comment-on-prs.yml

```diff
@@ -0,0 +1,77 @@
+name: Comment on Pull request
+#Cribbed from https://github.com/orgs/community/discussions/51403#discussioncomment-5535167
+
+on:
+  workflow_run:
+    workflows: [Test Opencast]
+    types:
+      - completed
+
+jobs:
+  comment:
+    if: github.event.workflow_run.conclusion == 'success'
```

Comment:
Does it make sense to limit this to the `opencast/opencast` repo?

Reply:
Fixed!